content/en/docs/getting-started/prerequisites/hardware/snp.md

```diff
@@ -8,5 +8,107 @@ tags:
 - SEV
 ---
 
-TODO
+## Platform Setup
+
+In order to launch SEV or SNP memory encrypted guests, the host must be prepared with a compatible kernel, `6.8.0-rc5-next-20240221-snp-host-cc2568386`. AMD custom changes and required components and repositories will eventually be upstreamed. 
+
+[Sev-utils](https://github.com/amd/sev-utils/blob/coco-202402240000/docs/snp.md) is an easy way to install the required host kernel, but it will unnecessarily build AMD compatible guest kernel, OVMF, and QEMU components. The additional components can be used with the script utility to test launch and attest a base QEMU SNP guest. However, for the CoCo use case, they are already packaged and delivered with Kata.
+
+Alternatively, refer to the [AMDESE guide](https://github.com/confidential-containers/amdese-amdsev/tree/amd-snp-202402240000?tab=readme-ov-file#prepare-host) to manually build the host kernel and other components.
```

Comment:
So what are we telling beginner users to do? You mention that sev-utils but what exactly are they supposed to do with that? You sort of seem to suggest that they shouldn't use it because it installs too much stuff. Should they be using the second option?

Reply:
I think I worded it wrong. What I had meant is that the coco tagged version of sev-utils cleans up some of the unnecessary things that it installs, but the second option is also a valid way of getting the right kernel.